sidebars.js

```diff
@@ -141,17 +141,36 @@ const sidebars = {
     },
     {
       type: "category",
-      label: "Customize",
+      label: "Customize your Subnet",
       collapsed: true,
       items: [
         "subnets/customize-a-subnet",
+        "subnets/cross-subnet-communication"
+      ],
+    },
+    {
+      type: "category",
+      label: "Build A Subnet-EVM Chain",
```

Comment:
```suggestion
      label: "Build a Subnet-EVM Chain",
```
If you are not sure, please utilize https://titlecase.com/